plasmapy/atomic/tests/test_particle_class.py

```diff
@@ -192,6 +200,7 @@
       'element_name': 'hydrogen',
       'isotope': 'D',
       'ionic_symbol': 'D 1+',
+      'roman_symbol': 'H II',
```

Comment:
I'm not sure this is correct... I would've thought this should be `D II` so that it is unambiguous.
@namurphy  thoughts?

Reply:
Yep, I forgot to mention isotopes in the description.  I brought this up above.